controllers/operandrequest/reconcile_operator.go

```diff
@@ -330,13 +331,17 @@ func (r *Reconciler) absentOperatorsAndOperands(ctx context.Context, requestInst
 					mu.Lock()
 					defer mu.Unlock()
 					merr.Add(err)
+					deepcopy.Remove(o)
```

Comment:
May I ask that, should we remove the `Operands` from `deepcopy` only after it is successfully deleted?

I understand that if there is a timeout, we should return the `deepcopy` that indicates the remaining `Operands`. But it should include the `Operands` that are not deleted and failed to delete.

Reply:
Yes, you're correct, let me update it.

Reply:
I am thinking separate the remaining operators/operands that are not deleted and failed to delete. 

I will update the error messages to make it's more clear to users.